lmdeploy/pytorch/backends/cuda/token_dispatcher.py

```diff
@@ -45,7 +45,7 @@ def get_buffer_common(
             num_nvl_bytes=num_nvl_bytes,
             num_rdma_bytes=num_rdma_bytes,
             low_latency_mode=True,
-            num_qps_per_rank=num_experts // group.size(),
+            num_qps_per_rank=max(num_experts // group.size(), Buffer.num_sms // 2),
```

Comment:
Since `Buffer` is only imported when `use_deepep==True`, we may try
```
num_qps_per_rank=max(num_experts // group.size(), Buffer.num_sms // 2) if use_deepep else num_experts // group.size()
```

Reply:
It will be left to another PR